docs/streams/developer-guide/config-streams.html

```diff
@@ -83,6 +83,7 @@
               <li><a class="reference internal" href="#probing-rebalance-interval-ms" id="id30">probing.rebalance.interval.ms</a></li>
               <li><a class="reference internal" href="#processing-exception-handler" id="id41">processing.exception.handler</a></li>
               <li><a class="reference internal" href="#processing-guarantee" id="id25">processing.guarantee</a></li>
+              <li><a class="reference internal" href="#production-exception-handler" id="id43">production.exception.handler</a></li>
```

Comment:
This is still adding a new link `<a ... href="#production-exception-handler" ...>` which is dangling...

I think we should update the HTML here to use `production-exception-handler` as new anchor point: https://kafka.apache.org/38/documentation/streams/developer-guide/config-streams.html#default-production-exception-handler